tests/test_stubs.py

```diff
@@ -16,7 +15,8 @@ def tmpdir():
 
 @contextmanager
 def tmplog(fname):
-    yield io.open(fname, mode='w')
+    file_ptr = open(fname, mode='w')
+    yield file_ptr
```

Comment:
Does this test still run even though you have the `yield`? I didn't think that was supported by pytest.

Reply:
yield works fine with pytest - it's used in a few places